lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service_model_base.rb

```diff
@@ -115,6 +123,7 @@ def self.dynamic_service_model_creation(ar_model, super_class)
       Class.new(super_class) do |klass|
         ::MiqAeMethodService.const_set(model_to_service_model_name(ar_model), klass)
         expose_class_attributes(klass)
+        expose_class_associations(klass)
```

Comment:
👍

Reply:
Based on https://github.com/ManageIQ/manageiq-automation_engine/pull/308#discussion_r267967934 we wanted to create a new method that both `inherited` and `dynamic_service_model_creation` used.  

If we need to add more class initialization logic in the future we are just forcing it to be added in two place now.

Reply:
yeah, if we are going to add more very soon, it might make sense to extract that logic to a method either now or when we do it.  I don't know if it's very soon but we've thought more was coming in the future and it never came so I'm not against holding off on extracting it to a method when we need a 3rd consumer.